pyswervedrive/module.py

```diff
@@ -87,11 +89,27 @@ def __init__(
         self.steer_motor.setInverted(self.reverse_steer_direction)
         # self.steer_motor.setSelectedSensorPosition(0)
 
-        self.steer_motor.config_kP(0, 1, 10)
-        self.steer_motor.config_kI(0, 0.0, 10)
-        self.steer_motor.config_kD(0, 0.02, 10)
+        # Set relevant frame periods to be at least as fast as periodic rate
+        self.steer_motor.setStatusFramePeriod(
+            ctre.WPI_TalonSRX.StatusFrameEnhanced.Status_13_Base_PIDF0, 10, 10
+        )
+        self.steer_motor.setStatusFramePeriod(
+            ctre.WPI_TalonSRX.StatusFrameEnhanced.Status_10_MotionMagic, 10, 10
+        )
+
         self.steer_motor.selectProfileSlot(0, 0)
-        self.steer_motor.config_kF(0, 0, 10)
+        self.steer_motor.config_kF(0, 1023 / self.STEER_FREE_SPEED, 10)
+        self.steer_motor.config_kP(0, 0.75, 10)
+        self.steer_motor.config_kI(0, 0.0, 10)
+        self.steer_motor.config_kD(0, 0.0, 10)
+        self.steer_motor.configMotionCruiseVelocity(2000, 10)
+        self.steer_motor.configMotionAcceleration(10000, 10)
+
+        self.steer_motor.configVoltageCompSaturation(12, timeoutMs=10)
+        self.steer_motor.configPeakCurrentLimit(10, timeoutMs=10)
```

Comment:
It doesn't really make sense to have a peak current limit equal to the continuous current limit.